ReactWindows/ReactNative/DevSupport/DevServerHelper.cs

```diff
@@ -221,5 +268,13 @@ private string CreateOnChangeEndpointUrl(string host)
         {
             return string.Format(CultureInfo.InvariantCulture, OnChangeEndpointUrlFormat, host);
         }
+
+        private string CreateLaunchDevToolsCommandUrl()
```

Comment:
Will not become public.

---
In reply to: [55510803](https://github.com/CatalystCode/react-native/pull/235#discussion_r55510803) [](ancestors = 55510803)